.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -104,6 +104,7 @@ if [ "$REPO" == "terraform-google-conversion" ]; then
     rm -rf ./tfplan2cai/testdata/templates/
     rm -rf ./tfplan2cai/testdata/generatedconvert/
     rm -rf ./tfplan2cai/converters/google/provider
+    rm -rf ./tfplan2cai/converters/google/resources/*
```

Comment:
I think this should just delete the entire `resources` folder

```suggestion
    rm -rf ./tfplan2cai/converters/google/resources
```